packages/fxa-settings/src/models/Account.ts

```diff
@@ -74,3 +74,10 @@ export function useAccount() {
   });
   return data!.account;
 }
+
+export function useRefetchableAccount() {
+  const { data, refetch } = useQuery<{ account: Account }>(GET_ACCOUNT, {
+    fetchPolicy: 'cache-first',
+  });
+  return { account: data!.account, refetchAccount: refetch };
+}
```

Comment:
I'm not sure this is necessary. Mutations can specify an `update` function or `refetchQueries` to update the cache. https://www.apollographql.com/docs/react/api/react/hooks/#options-2

I also discovered in #6276 that `useQuery` doesn't play nice with the router.

Reply:
Oh nice. Will update. And I'll temporarily add [this](https://github.com/mozilla/fxa/pull/6276/files#diff-5dfa9e3125fdc91e2fae5233ce23fbbeR72) here as well, in case this is merged first.